pkg/inventory/inventorycm.go

```diff
@@ -94,7 +95,7 @@ func (icm *InventoryConfigMap) Load() (object.ObjMetadataSet, error) {
 // Store is an Inventory interface function implemented to store
 // the object metadata in the wrapped ConfigMap. Actual storing
 // happens in "GetObject".
-func (icm *InventoryConfigMap) Store(objMetas object.ObjMetadataSet) error {
+func (icm *InventoryConfigMap) Store(objMetas object.ObjMetadataSet, _ map[object.ObjMetadata]status.ApplyReconcileStatus) error {
```

Comment:
The fact that we don't even store the apply status in the default `ConfigMap` object illustrates that this functionality might not belong in the `cli-utils`. Are we sure we want to do this?

Reply:
We need to improve the interface for the inventory client and the inventory object later: getting the inventory object, updating the inventory object locally or in the cluster will be covered by the interface. When we do that, the `Store` function may not be needed. Changing this function here is to make minimal change on the existing code before we refactor the interfaces.